modules/videopress/class.videopress-player.php

```diff
@@ -597,7 +597,16 @@ private function get_flash_parameters() {
 		if ( ! isset( $this->video->players->swf->params ) )
 			return array();
 		else
-			return self::esc_flash_params( apply_filters( 'video_flash_params', (array) $this->video->players->swf->params, 10, 1 ) );
+			return self::esc_flash_params(
+				/**
+				 * Filters the Flash parameters of the VideoPress player.
```

Comment:
Ah yes. Forgot about that. That works for me. I wouldn't want to add them now until we're sure what VP will look like in JP.